src/protocol.ts

```diff
@@ -577,6 +577,10 @@ export namespace AgentJobHistoryRequest {
 	export const type = new RequestType<types.AgentJobHistoryParams, types.AgentJobHistoryResult, void, void>('agent/jobhistory');
 }
 
+export namespace AgentJobActionRequest {
+	export const type = new RequestType<types.AgentJobActionParams, types.AgentJobActionResult, void, void>('agent/jobaction');
```

Comment:
not sure if it makes more sense to have distinct start/stop event.  In any case, I want a build tonight with all the pieces put together so let's take as-is.